parameters/world_modeling/world_model.yaml

```diff
@@ -19,13 +19,18 @@ plugins:
 
 - name: navigation
   lib: libed_navigation_plugin.so
+  frequency: 1
```

Comment:
I hope/guess this is interpreted correctly as a double?

Reply:
Yep, working